src/transformers/models/deberta/modeling_deberta.py

```diff
@@ -773,6 +775,18 @@ def _init_weights(self, module):
         if isinstance(module, nn.Linear) and module.bias is not None:
             module.bias.data.zero_()
 
+    def _pre_load_hook(self, state_dict, prefix, local_metadata, strict, missing_keys, unexpected_keys, error_msgs):
```

Comment:
Can we add some more explanation here on what this does. Also, we are sure that this doesn't have any backward breaking changes for deberta_v1 no?